martus-client/source/org/martus/client/test/TestSha1ToSha2Change.java

```diff
@@ -229,6 +246,42 @@ private MockBulletinStore changeBulletinStoreSecurity(MockBulletinStore oldStore
 		return new MockBulletinStore(oldStore.getStoreRootDir(), oldStore.getWriteableDatabase(), newSecurity);
 	}
 
+	public void testParseRetrieveCommandBundleSignedWithSha1UsingSha2Verifier() throws Exception
+	{
+		Vector uids = new Vector();
+		uids.add(UniversalIdForTesting.createDummyUniversalId());
+		uids.add(UniversalIdForTesting.createDummyUniversalId());
+		RetrieveCommand rc = new RetrieveCommand("folder", uids);
+		byte[] bundle = appSha1.createRetrieveCommandBundle(rc);
+
```

Comment:
The creation of the bundle udinh uids can be extracted out into a helper method.